Index: scroom-0.1.1/debian/control
===================================================================
--- scroom-0.1.1.orig/debian/control	2010-11-21 13:51:59.672600664 +0100
+++ scroom-0.1.1/debian/control	2010-11-21 13:52:14.142114333 +0100
@@ -2,7 +2,7 @@
 Section: universe/graphics
 Priority: extra
 Maintainer: Kees-Jan Dijkzeul <kees-jan.dijkzeul@iae.nl>
-Build-Depends: debhelper (>= 7), autotools-dev, pkg-config, libboost-dev, libboost-thread-dev, libboost-system-dev, libtiff4-dev, libgtk2.0-dev, libglib2.0-dev, libglade2-dev, libcairo2-dev
+Build-Depends: debhelper (>= 7), autotools-dev, pkg-config, libtiff4-dev, libgtk2.0-dev, libglib2.0-dev, libglade2-dev, libcairo2-dev
 Standards-Version: 3.8.3
 Homepage: https://github.com/kees-jan/scroom/wiki
 
